File: js/dragonkinNames.js
var nm1 = ["","","","","","b","br","dr","g","gr","h","k","kr","m","n","r","s","sr","str","t","tr","v","z"];
var nm2 = ["a","e","i","o","u","a","e","i","o","u","a","e","i","o","u","a","e","i","o","u","a","e","i","o","u","a","e","i","o","u","a","e","i","o","u","ai","ae","ia","iu","io","eo"];
var nm3 = ["cr","cg","cn","csh","cd","cdr","dr","dg","dgr","dk","dkr","k","kr","kt","kth","ksh","l","lk","lt","ldr","lg","lgr","lsh","lz","n","nd","ndr","nsh","nsk","r","rc","rph","rsh","rth","rd","rdr","rgr","rg","rz","rzr","rsh","s","sth","shk","sk","sg","skr","th","tr","tr","tg","z","zz","zg","zk"];
var nm4 = ["b","d","g","j","k","l","n","r","s","sh","z"];
var nm5 = ["","","","c","d","g","gg","k","ks","n","nd","ph","s","th","x","z"];

var nm6 = ["","","","","","","","","","","b","bh","c","ch","d","g","h","kh","l","m","n","ph","phr","r","s","shr","str","sth","t","th","tr","z","zh"];
var nm7 = ["a","e","i","o","u","a","e","i","o","u","a","e","i","o","u","a","e","i","o","u","a","e","i","o","u","a","e","i","o","u","a","e","i","o","u","ai","ae","ia","ea","ie","ei"];
var nm8 = ["dr","dh","dn","dhr","gn","gr","ghr","gtr","gt","k","kk","kh","kt","kth","l","lk","ll","lg","ld","ldr","lgr","ln","lm","lkh","ls","lz","n","nd","ndh","ndr","ns","nsh","nz","nh","nhr","ng","ngh","r","rc","rph","rsh","rz","rl","s","sh","ss","sth","sht","shl","sn","sg","sk","th","thr","thn","tr","z","zh"];
var nm9 = ["l","m","n","r","s","sh","t","th","x","z"];
var nm10 = ["","","","","","","","","","","h","s","sh","th","x","z"];

var nm11 = ["","","","","","","","b","ch","d","g","h","k","kr","l","m","n","r","s","sr","str","sth","t","tr","th","z"];
var nm12 = ["a","e","i","o","u","a","e","i","o","u","a","e","i","o","u","a","e","i","o","u","a","e","i","o","u","a","e","i","o","u","a","e","i","o","u","ai","ae","ia","ea","io","ie"];
var nm13 = ["cr","cn","cd","dr","dh","dg","dhr","gn","gr","ghr","k","kk","kt","kth","l","lk","ll","lg","ld","ldr","lgr","lz","n","nd","ndr","ns","nsh","nz","ng","r","rc","rph","rsh","rz","rd","rdr","rgr","rg","s","sh","ss","sth","sht","sth","sn","sg","sk","th","thr","tr","z","zg","zh"];
var nm14 = ["b","d","g","l","n","r","s","sh","t","th","z"];
var nm15 = ["","","","h","n","s","sh","t","th","x","z"];

var br = "";

function nameGen(type){
	$('#placeholder').css('textTransform', 'capitalize');
	var tp = type;
	var element = document.createElement("div");
	element.setAttribute("id", "result");
	
	for(i = 0; i < 10; i++){
		if(tp === 1){
			rnd = Math.floor(Math.random() * nm6.length);
			rnd2 = Math.floor(Math.random() * nm7.length);
			rnd3 = Math.floor(Math.random() * nm8.length);
			rnd4 = Math.floor(Math.random() * nm7.length);
			rnd5 = Math.floor(Math.random() * nm10.length);
			while(nm8[rnd3] === nm6[rnd] || nm8[rnd3] === nm10[rnd5]){
				rnd3 = Math.floor(Math.random() * nm8.length);
			}
			if(i < 6){
				names = nm6[rnd] + nm7[rnd2] + nm8[rnd3] + nm7[rnd4] + nm10[rnd5];
			}else{
				rnd6 = Math.floor(Math.random() * nm9.length);
				rnd7 = Math.floor(Math.random() * nm7.length);
				while(nm8[rnd3] === nm9[rnd6] || nm9[rnd6] === nm10[rnd5]){
					rnd6 = Math.floor(Math.random() * nm9.length);
				}
				if(i < 8){
					names = nm6[rnd] + nm7[rnd2] + nm8[rnd3] + nm7[rnd4] + nm9[rnd6] + nm7[rnd7] + nm10[rnd5];
				}else{
					names = nm6[rnd] + nm7[rnd2] + nm9[rnd6] + nm7[rnd7] + nm8[rnd3] + nm7[rnd4] + nm10[rnd5];
				}
			}
		}else if(tp === 2){
			rnd = Math.floor(Math.random() * nm11.length);
			rnd2 = Math.floor(Math.random() * nm12.length);
			rnd3 = Math.floor(Math.random() * nm13.length);
			rnd4 = Math.floor(Math.random() * nm12.length);
			rnd5 = Math.floor(Math.random() * nm15.length);
			while(nm13[rnd3] === nm11[rnd] || nm13[rnd3] === nm15[rnd5]){
				rnd3 = Math.floor(Math.random() * nm13.length);
			}
			if(i < 6){
				names = nm11[rnd] + nm12[rnd2] + nm13[rnd3] + nm12[rnd4] + nm15[rnd5];
			}else{
				rnd6 = Math.floor(Math.random() * nm14.length);
				rnd7 = Math.floor(Math.random() * nm12.length);
				while(nm13[rnd3] === nm14[rnd6] || nm14[rnd6] === nm15[rnd5]){
					rnd6 = Math.floor(Math.random() * nm14.length);
				}
				if(i < 8){
					names = nm11[rnd] + nm12[rnd2] + nm13[rnd3] + nm12[rnd4] + nm14[rnd6] + nm12[rnd7] + nm15[rnd5];
				}else{
					names = nm11[rnd] + nm12[rnd2] + nm14[rnd6] + nm12[rnd7] + nm13[rnd3] + nm12[rnd4] + nm15[rnd5];
				}
			}
		}else{
			rnd = Math.floor(Math.random() * nm1.length);
			rnd2 = Math.floor(Math.random() * nm2.length);
			rnd3 = Math.floor(Math.random() * nm3.length);
			rnd4 = Math.floor(Math.random() * nm2.length);
			rnd5 = Math.floor(Math.random() * nm5.length);
			while(nm3[rnd3] === nm1[rnd] || nm3[rnd3] === nm5[rnd5]){
				rnd3 = Math.floor(Math.random() * nm3.length);
			}
			if(i < 6){
				names = nm1[rnd] + nm2[rnd2] + nm3[rnd3] + nm2[rnd4] + nm5[rnd5];
			}else{
				rnd6 = Math.floor(Math.random() * nm4.length);
				rnd7 = Math.floor(Math.random() * nm2.length);
				while(nm3[rnd3] === nm4[rnd6] || nm4[rnd6] === nm5[rnd5]){
					rnd6 = Math.floor(Math.random() * nm4.length);
				}
				if(i < 8){
					names = nm1[rnd] + nm2[rnd2] + nm3[rnd3] + nm2[rnd4] + nm4[rnd6] + nm2[rnd7] + nm5[rnd5];
				}else{
					names = nm1[rnd] + nm2[rnd2] + nm4[rnd6] + nm2[rnd7] + nm3[rnd3] + nm2[rnd4] + nm5[rnd5];
				}
			}
		}
		br = document.createElement('br');	
		element.appendChild(document.createTextNode(names));
		element.appendChild(br);
	}
	if(document.getElementById("result")){
		document.getElementById("placeholder").removeChild(document.getElementById("result"));
	}		
	document.getElementById("placeholder").appendChild(element);
}

File: generators/dragonkinNames.py
__all__ = ['dragonkinNames']

# Don't look below, you will not understand this Python code :) I don't.

from js2py.pyjs import *
# setting scope
var = Scope( JS_BUILTINS )
set_global_object(var)

# Code follows:
var.registers(['nm1', 'nm11', 'nm12', 'nm4', 'nameGen', 'nm5', 'nm8', 'nm14', 'nm3', 'nm13', 'nm7', 'nm9', 'nm10', 'nm2', 'nm15', 'nm6'])
@Js
def PyJsHoisted_nameGen_(type, this, arguments, var=var):
    var = Scope({'arguments':arguments, 'this':this, 'type':type}, var)
    var.registers(['tp', 'result', 'type'])
    var.put('tp', var.get('type'))
    var.put('result', Js([]))
    #for JS loop
    var.put('i', Js(0.0))
    while (var.get('i')<Js(10.0)):
        try:
            if PyJsStrictEq(var.get('tp'),Js(1.0)):
                var.put('rnd', var.get('Math').callprop('floor', (var.get('Math').callprop('random')*var.get('nm6').get('length'))))
                var.put('rnd2', var.get('Math').callprop('floor', (var.get('Math').callprop('random')*var.get('nm7').get('length'))))
                var.put('rnd3', var.get('Math').callprop('floor', (var.get('Math').callprop('random')*var.get('nm8').get('length'))))
                var.put('rnd4', var.get('Math').callprop('floor', (var.get('Math').callprop('random')*var.get('nm7').get('length'))))
                var.put('rnd5', var.get('Math').callprop('floor', (var.get('Math').callprop('random')*var.get('nm10').get('length'))))
                while (PyJsStrictEq(var.get('nm8').get(var.get('rnd3')),var.get('nm6').get(var.get('rnd'))) or PyJsStrictEq(var.get('nm8').get(var.get('rnd3')),var.get('nm10').get(var.get('rnd5')))):
                    var.put('rnd3', var.get('Math').callprop('floor', (var.get('Math').callprop('random')*var.get('nm8').get('length'))))
                if (var.get('i')<Js(6.0)):
                    var.put('names', ((((var.get('nm6').get(var.get('rnd'))+var.get('nm7').get(var.get('rnd2')))+var.get('nm8').get(var.get('rnd3')))+var.get('nm7').get(var.get('rnd4')))+var.get('nm10').get(var.get('rnd5'))))
                else:
                    var.put('rnd6', var.get('Math').callprop('floor', (var.get('Math').callprop('random')*var.get('nm9').get('length'))))
                    var.put('rnd7', var.get('Math').callprop('floor', (var.get('Math').callprop('random')*var.get('nm7').get('length'))))
                    while (PyJsStrictEq(var.get('nm8').get(var.get('rnd3')),var.get('nm9').get(var.get('rnd6'))) or PyJsStrictEq(var.get('nm9').get(var.get('rnd6')),var.get('nm10').get(var.get('rnd5')))):
                        var.put('rnd6', var.get('Math').callprop('floor', (var.get('Math').callprop('random')*var.get('nm9').get('length'))))
                    if (var.get('i')<Js(8.0)):
                        var.put('names', ((((((var.get('nm6').get(var.get('rnd'))+var.get('nm7').get(var.get('rnd2')))+var.get('nm8').get(var.get('rnd3')))+var.get('nm7').get(var.get('rnd4')))+var.get('nm9').get(var.get('rnd6')))+var.get('nm7').get(var.get('rnd7')))+var.get('nm10').get(var.get('rnd5'))))
                    else:
                        var.put('names', ((((((var.get('nm6').get(var.get('rnd'))+var.get('nm7').get(var.get('rnd2')))+var.get('nm9').get(var.get('rnd6')))+var.get('nm7').get(var.get('rnd7')))+var.get('nm8').get(var.get('rnd3')))+var.get('nm7').get(var.get('rnd4')))+var.get('nm10').get(var.get('rnd5'))))
            else:
                if PyJsStrictEq(var.get('tp'),Js(2.0)):
                    var.put('rnd', var.get('Math').callprop('floor', (var.get('Math').callprop('random')*var.get('nm11').get('length'))))
                    var.put('rnd2', var.get('Math').callprop('floor', (var.get('Math').callprop('random')*var.get('nm12').get('length'))))
                    var.put('rnd3', var.get('Math').callprop('floor', (var.get('Math').callprop('random')*var.get('nm13').get('length'))))
                    var.put('rnd4', var.get('Math').callprop('floor', (var.get('Math').callprop('random')*var.get('nm12').get('length'))))
                    var.put('rnd5', var.get('Math').callprop('floor', (var.get('Math').callprop('random')*var.get('nm15').get('length'))))
                    while (PyJsStrictEq(var.get('nm13').get(var.get('rnd3')),var.get('nm11').get(var.get('rnd'))) or PyJsStrictEq(var.get('nm13').get(var.get('rnd3')),var.get('nm15').get(var.get('rnd5')))):
                        var.put('rnd3', var.get('Math').callprop('floor', (var.get('Math').callprop('random')*var.get('nm13').get('length'))))
                    if (var.get('i')<Js(6.0)):
                        var.put('names', ((((var.get('nm11').get(var.get('rnd'))+var.get('nm12').get(var.get('rnd2')))+var.get('nm13').get(var.get('rnd3')))+var.get('nm12').get(var.get('rnd4')))+var.get('nm15').get(var.get('rnd5'))))
                    else:
                        var.put('rnd6', var.get('Math').callprop('floor', (var.get('Math').callprop('random')*var.get('nm14').get('length'))))
                        var.put('rnd7', var.get('Math').callprop('floor', (var.get('Math').callprop('random')*var.get('nm12').get('length'))))
                        while (PyJsStrictEq(var.get('nm13').get(var.get('rnd3')),var.get('nm14').get(var.get('rnd6'))) or PyJsStrictEq(var.get('nm14').get(var.get('rnd6')),var.get('nm15').get(var.get('rnd5')))):
                            var.put('rnd6', var.get('Math').callprop('floor', (var.get('Math').callprop('random')*var.get('nm14').get('length'))))
                        if (var.get('i')<Js(8.0)):
                            var.put('names', ((((((var.get('nm11').get(var.get('rnd'))+var.get('nm12').get(var.get('rnd2')))+var.get('nm13').get(var.get('rnd3')))+var.get('nm12').get(var.get('rnd4')))+var.get('nm14').get(var.get('rnd6')))+var.get('nm12').get(var.get('rnd7')))+var.get('nm15').get(var.get('rnd5'))))
                        else:
                            var.put('names', ((((((var.get('nm11').get(var.get('rnd'))+var.get('nm12').get(var.get('rnd2')))+var.get('nm14').get(var.get('rnd6')))+var.get('nm12').get(var.get('rnd7')))+var.get('nm13').get(var.get('rnd3')))+var.get('nm12').get(var.get('rnd4')))+var.get('nm15').get(var.get('rnd5'))))
                else:
                    var.put('rnd', var.get('Math').callprop('floor', (var.get('Math').callprop('random')*var.get('nm1').get('length'))))
                    var.put('rnd2', var.get('Math').callprop('floor', (var.get('Math').callprop('random')*var.get('nm2').get('length'))))
                    var.put('rnd3', var.get('Math').callprop('floor', (var.get('Math').callprop('random')*var.get('nm3').get('length'))))
                    var.put('rnd4', var.get('Math').callprop('floor', (var.get('Math').callprop('random')*var.get('nm2').get('length'))))
                    var.put('rnd5', var.get('Math').callprop('floor', (var.get('Math').callprop('random')*var.get('nm5').get('length'))))
                    while (PyJsStrictEq(var.get('nm3').get(var.get('rnd3')),var.get('nm1').get(var.get('rnd'))) or PyJsStrictEq(var.get('nm3').get(var.get('rnd3')),var.get('nm5').get(var.get('rnd5')))):
                        var.put('rnd3', var.get('Math').callprop('floor', (var.get('Math').callprop('random')*var.get('nm3').get('length'))))
                    if (var.get('i')<Js(6.0)):
                        var.put('names', ((((var.get('nm1').get(var.get('rnd'))+var.get('nm2').get(var.get('rnd2')))+var.get('nm3').get(var.get('rnd3')))+var.get('nm2').get(var.get('rnd4')))+var.get('nm5').get(var.get('rnd5'))))
                    else:
                        var.put('rnd6', var.get('Math').callprop('floor', (var.get('Math').callprop('random')*var.get('nm4').get('length'))))
                        var.put('rnd7', var.get('Math').callprop('floor', (var.get('Math').callprop('random')*var.get('nm2').get('length'))))
                        while (PyJsStrictEq(var.get('nm3').get(var.get('rnd3')),var.get('nm4').get(var.get('rnd6'))) or PyJsStrictEq(var.get('nm4').get(var.get('rnd6')),var.get('nm5').get(var.get('rnd5')))):
                            var.put('rnd6', var.get('Math').callprop('floor', (var.get('Math').callprop('random')*var.get('nm4').get('length'))))
                        if (var.get('i')<Js(8.0)):
                            var.put('names', ((((((var.get('nm1').get(var.get('rnd'))+var.get('nm2').get(var.get('rnd2')))+var.get('nm3').get(var.get('rnd3')))+var.get('nm2').get(var.get('rnd4')))+var.get('nm4').get(var.get('rnd6')))+var.get('nm2').get(var.get('rnd7')))+var.get('nm5').get(var.get('rnd5'))))
                        else:
                            var.put('names', ((((((var.get('nm1').get(var.get('rnd'))+var.get('nm2').get(var.get('rnd2')))+var.get('nm4').get(var.get('rnd6')))+var.get('nm2').get(var.get('rnd7')))+var.get('nm3').get(var.get('rnd3')))+var.get('nm2').get(var.get('rnd4')))+var.get('nm5').get(var.get('rnd5'))))
            var.get('result').callprop('push', var.get('names'))
        finally:
                (var.put('i',Js(var.get('i').to_number())+Js(1))-Js(1))
    pass
    return var.get('result')
PyJsHoisted_nameGen_.func_name = 'nameGen'
var.put('nameGen', PyJsHoisted_nameGen_)
var.put('nm1', Js([Js(''), Js(''), Js(''), Js(''), Js(''), Js('b'), Js('br'), Js('dr'), Js('g'), Js('gr'), Js('h'), Js('k'), Js('kr'), Js('m'), Js('n'), Js('r'), Js('s'), Js('sr'), Js('str'), Js('t'), Js('tr'), Js('v'), Js('z')]))
var.put('nm2', Js([Js('a'), Js('e'), Js('i'), Js('o'), Js('u'), Js('a'), Js('e'), Js('i'), Js('o'), Js('u'), Js('a'), Js('e'), Js('i'), Js('o'), Js('u'), Js('a'), Js('e'), Js('i'), Js('o'), Js('u'), Js('a'), Js('e'), Js('i'), Js('o'), Js('u'), Js('a'), Js('e'), Js('i'), Js('o'), Js('u'), Js('a'), Js('e'), Js('i'), Js('o'), Js('u'), Js('ai'), Js('ae'), Js('ia'), Js('iu'), Js('io'), Js('eo')]))
var.put('nm3', Js([Js('cr'), Js('cg'), Js('cn'), Js('csh'), Js('cd'), Js('cdr'), Js('dr'), Js('dg'), Js('dgr'), Js('dk'), Js('dkr'), Js('k'), Js('kr'), Js('kt'), Js('kth'), Js('ksh'), Js('l'), Js('lk'), Js('lt'), Js('ldr'), Js('lg'), Js('lgr'), Js('lsh'), Js('lz'), Js('n'), Js('nd'), Js('ndr'), Js('nsh'), Js('nsk'), Js('r'), Js('rc'), Js('rph'), Js('rsh'), Js('rth'), Js('rd'), Js('rdr'), Js('rgr'), Js('rg'), Js('rz'), Js('rzr'), Js('rsh'), Js('s'), Js('sth'), Js('shk'), Js('sk'), Js('sg'), Js('skr'), Js('th'), Js('tr'), Js('tr'), Js('tg'), Js('z'), Js('zz'), Js('zg'), Js('zk')]))
var.put('nm4', Js([Js('b'), Js('d'), Js('g'), Js('j'), Js('k'), Js('l'), Js('n'), Js('r'), Js('s'), Js('sh'), Js('z')]))
var.put('nm5', Js([Js(''), Js(''), Js(''), Js('c'), Js('d'), Js('g'), Js('gg'), Js('k'), Js('ks'), Js('n'), Js('nd'), Js('ph'), Js('s'), Js('th'), Js('x'), Js('z')]))
var.put('nm6', Js([Js(''), Js(''), Js(''), Js(''), Js(''), Js(''), Js(''), Js(''), Js(''), Js(''), Js('b'), Js('bh'), Js('c'), Js('ch'), Js('d'), Js('g'), Js('h'), Js('kh'), Js('l'), Js('m'), Js('n'), Js('ph'), Js('phr'), Js('r'), Js('s'), Js('shr'), Js('str'), Js('sth'), Js('t'), Js('th'), Js('tr'), Js('z'), Js('zh')]))
var.put('nm7', Js([Js('a'), Js('e'), Js('i'), Js('o'), Js('u'), Js('a'), Js('e'), Js('i'), Js('o'), Js('u'), Js('a'), Js('e'), Js('i'), Js('o'), Js('u'), Js('a'), Js('e'), Js('i'), Js('o'), Js('u'), Js('a'), Js('e'), Js('i'), Js('o'), Js('u'), Js('a'), Js('e'), Js('i'), Js('o'), Js('u'), Js('a'), Js('e'), Js('i'), Js('o'), Js('u'), Js('ai'), Js('ae'), Js('ia'), Js('ea'), Js('ie'), Js('ei')]))
var.put('nm8', Js([Js('dr'), Js('dh'), Js('dn'), Js('dhr'), Js('gn'), Js('gr'), Js('ghr'), Js('gtr'), Js('gt'), Js('k'), Js('kk'), Js('kh'), Js('kt'), Js('kth'), Js('l'), Js('lk'), Js('ll'), Js('lg'), Js('ld'), Js('ldr'), Js('lgr'), Js('ln'), Js('lm'), Js('lkh'), Js('ls'), Js('lz'), Js('n'), Js('nd'), Js('ndh'), Js('ndr'), Js('ns'), Js('nsh'), Js('nz'), Js('nh'), Js('nhr'), Js('ng'), Js('ngh'), Js('r'), Js('rc'), Js('rph'), Js('rsh'), Js('rz'), Js('rl'), Js('s'), Js('sh'), Js('ss'), Js('sth'), Js('sht'), Js('shl'), Js('sn'), Js('sg'), Js('sk'), Js('th'), Js('thr'), Js('thn'), Js('tr'), Js('z'), Js('zh')]))
var.put('nm9', Js([Js('l'), Js('m'), Js('n'), Js('r'), Js('s'), Js('sh'), Js('t'), Js('th'), Js('x'), Js('z')]))
var.put('nm10', Js([Js(''), Js(''), Js(''), Js(''), Js(''), Js(''), Js(''), Js(''), Js(''), Js(''), Js('h'), Js('s'), Js('sh'), Js('th'), Js('x'), Js('z')]))
var.put('nm11', Js([Js(''), Js(''), Js(''), Js(''), Js(''), Js(''), Js(''), Js('b'), Js('ch'), Js('d'), Js('g'), Js('h'), Js('k'), Js('kr'), Js('l'), Js('m'), Js('n'), Js('r'), Js('s'), Js('sr'), Js('str'), Js('sth'), Js('t'), Js('tr'), Js('th'), Js('z')]))
var.put('nm12', Js([Js('a'), Js('e'), Js('i'), Js('o'), Js('u'), Js('a'), Js('e'), Js('i'), Js('o'), Js('u'), Js('a'), Js('e'), Js('i'), Js('o'), Js('u'), Js('a'), Js('e'), Js('i'), Js('o'), Js('u'), Js('a'), Js('e'), Js('i'), Js('o'), Js('u'), Js('a'), Js('e'), Js('i'), Js('o'), Js('u'), Js('a'), Js('e'), Js('i'), Js('o'), Js('u'), Js('ai'), Js('ae'), Js('ia'), Js('ea'), Js('io'), Js('ie')]))
var.put('nm13', Js([Js('cr'), Js('cn'), Js('cd'), Js('dr'), Js('dh'), Js('dg'), Js('dhr'), Js('gn'), Js('gr'), Js('ghr'), Js('k'), Js('kk'), Js('kt'), Js('kth'), Js('l'), Js('lk'), Js('ll'), Js('lg'), Js('ld'), Js('ldr'), Js('lgr'), Js('lz'), Js('n'), Js('nd'), Js('ndr'), Js('ns'), Js('nsh'), Js('nz'), Js('ng'), Js('r'), Js('rc'), Js('rph'), Js('rsh'), Js('rz'), Js('rd'), Js('rdr'), Js('rgr'), Js('rg'), Js('s'), Js('sh'), Js('ss'), Js('sth'), Js('sht'), Js('sth'), Js('sn'), Js('sg'), Js('sk'), Js('th'), Js('thr'), Js('tr'), Js('z'), Js('zg'), Js('zh')]))
var.put('nm14', Js([Js('b'), Js('d'), Js('g'), Js('l'), Js('n'), Js('r'), Js('s'), Js('sh'), Js('t'), Js('th'), Js('z')]))
var.put('nm15', Js([Js(''), Js(''), Js(''), Js('h'), Js('n'), Js('s'), Js('sh'), Js('t'), Js('th'), Js('x'), Js('z')]))
pass
pass


# Add lib to the module scope
dragonkinNames = var.to_python()